File: runtime-testsuite/test/org/antlr/v5/test/runtime/java/helpers/RuntimeTestLexer.java
package org.antlr.v5.test.runtime.java.helpers;

import org.antlr.v5.runtime.core.CharStream;
import org.antlr.v5.runtime.core.Lexer;

public abstract class RuntimeTestLexer extends Lexer {
	protected java.io.PrintStream outStream = System.out;

	public RuntimeTestLexer(CharStream input) { super(input); }

	public void setOutStream(java.io.PrintStream outStream) { this.outStream = outStream; }
}

File: runtime-testsuite/test/org/antlr/v5/test/runtime/kotlin/helpers/RuntimeTestLexer.kt
package org.antlr.v5.test.runtime.kotlin.helpers

import org.antlr.v5.runtime.core.CharStream
import org.antlr.v5.runtime.core.Lexer
import java.io.PrintStream

abstract class RuntimeTestLexer(input: CharStream) : Lexer(input) {
    var outStream: PrintStream = System.out
}
